tests.py

```diff
@@ -14,15 +14,16 @@
 
 if __name__ == '__main__':
 
+    import dateutil.parser
```

Comment:
It's good to get rid of these in Matplotlib itself, but it's hacks like these that make me a bit wary to keep this test setup around. Maybe when we figure out the cron jobs, we can run this on a nightly build or something.